src/Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensSchema.cs

```diff
@@ -125,5 +125,12 @@ public static SemanticTokensSchema LegacyTokenSchemaForRazor
 
         public static SemanticTokensSchema LegacyTokensSchemaForLSIF
             => s_vsTokenSchema;
+
+        public static string[] TokenModifiers = new string[]
+        {
+            // This must be in the same order as SemanticTokens.TokenModifiers, but skip the "None" item
+            SemanticTokenModifiers.Static,
+            nameof(SemanticTokens.TokenModifiers.ReassignedVariable)
```

Comment:
why nameof for reassignedvariable but not the other?

Reply:
`Static` is coming from the LSP protocol DLL, as its part of the standard set. I could change it to `nameof(SemanticTokens.TokenModifiers.Static)` for consistency, but `ReassignedVariable` can't come from anywhere else, as its entirely a Roslyn construct. Between Roslyn, Razor and VS, I don't think anyone actually implements this part of the spec properly (or nobody knows how it should be implemented - myself included!)